storage/streams/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java

```diff
@@ -0,0 +1,67 @@
+package io.apicurio.registry.storage.util;
```

Comment:
Is this class exactly the same as the one with the same name in the `kafka` module?  If yes can we have just one and put it somewhere that both modules can reach it?

Reply:
yes that makes sense, I'll do it

Reply:
I'm having issues moving it to a different module, because it depends on quarkus classes and because of this class being under src/test

Reply:
Yeah I wasn't sure how easy it would be to re-use that class.  If it's a pain, then don't worry about it.